multinode-demo/fullnode.sh

```diff
@@ -106,9 +88,25 @@ ledger_not_setup() {
   exit 1
 }
 
+setup_replicator_account() {
+  declare entrypoint_ip=$1
+  declare node_id_path=$2
+  declare stake=$3
+
+  if [[ -f "$node_id_path".configured ]]; then
+    echo "Replicator account has already been configured"
+  else
+    airdrop "$node_id_path" "$entrypoint_ip" "$stake" || return $?
+    touch "$node_id_path".configured
+  fi
+
+  return 0
+}
+
 args=()
 bootstrap_leader=false
```

Comment:
Let's replace the `bootstrap_leader` and `replicator` booleans with a single `node_type` string, to prevent future 🍝 from developping:
```bash
node_type=validator
...
if [[ $node_type = replicator ]]; then
  echo imma replicator
elif  [[ $node_type = bootstrap_leader ]]; then
  echo bs leader
fi
```

Reply:
done :) Looks much better.